lib/shared/src/misc/rpc/rpc.ts

```diff
@@ -341,10 +342,14 @@ export function addMessageListenersForExtensionAPI(
     }
 }
 
-const LOG_RPC_MESSAGES = true
+const LOG_RPC_MESSAGES = process.env.CODY_DEBUG === 'true'
```

Comment:
```suggestion
const LOG_RPC_MESSAGES = process.env.CODY_DEBUG === 'true'
```

@olafurpg Is it ok to use CODY_DEBUG or do you think this should have a separated environment variable?

Reply:
I recommend using a more specific name, something like `CODY_LOG_WEBVIEW_RPC_MESSAGES`. We should generally avoid reusing environment variables for different use-cases.

Reply:
Makes sense! Will keep that in mind, thanks Olaf <3